mmv1/third_party/terraform/services/container/node_config.go.erb

```diff
@@ -72,7 +72,7 @@ func schemaNodeConfig() *schema.Schema {
 					Type:         schema.TypeInt,
 					Optional:     true,
 					Computed:     true,
-					ForceNew:     true,
+					ForceNew:     false,
```

Comment:
removing the line entirely is the same as a `false` value, and that would be preferable to keep it in line with most other resources.